client/blocks/plugins-update-manager/schedule-form.tsx

```diff
@@ -133,9 +135,12 @@ export const ScheduleForm = ( props: Props ) => {
 		};
 
 		if ( formValid ) {
-			scheduleForEdit
-				? editUpdateSchedule( scheduleForEdit.id, params )
-				: createUpdateSchedule( params );
+			if ( scheduleForEdit ) {
+				editUpdateSchedule( scheduleForEdit.id, params );
+			} else {
+				createUpdateSchedule( params );
+				createMonitor( siteSlug );
```

Comment:
How are we sure that the creating schedule action is successful?
Let's create a monitor only if the updates are scheduled.

Reply:
Makes sense, thanks. Moved.